example/functions/create-session.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * This function creates a Stripe Checkout session and returns the session ID
+ * for use with Stripe.js (specifically the redirectToCheckout method).
+ *
+ * @see https://stripe.com/docs/payments/checkout/one-time
+ */
+
+const stripe = require('stripe')(process.env.REACT_APP_STRIPE_API_SECRET);
+
+/*
+ * Product data can be loaded from anywhere. In this case, we’re loading it from
+ * a local JSON file, but this could also come from an async call to your
+ * inventory management service, a database query, or some other API call.
+ *
+ * The important thing is that the product info is loaded from somewhere trusted
+ * so you know the pricing information is accurate.
+ */
+const inventory = require('./data/products.json');
+// const { validateCartItems } = useStripeCart;
+
+const validateCartItems = (inventorySrc, cartItems) => {
+  const validatedItems = Object.keys(cartItems).map((item) => {
+    const product = cartItems[item];
+    const validatedItem = inventorySrc.find(
+      (p) => p.name === cartItems[item].name
+    );
+    return {
+      name: validatedItem.name,
+      amount: validatedItem.price * product.quantity,
+      currency: validatedItem.currency,
+      quantity: product.quantity,
+    };
+  });
+
+  return validatedItems;
```

Comment:
Would it be cleaner to make this a return without using the `validatedItems` variable at all or does having the item make it clearer what is going on?

Reply:
I think having the item is more for communicating that these items are now valid after the check.